src/LocalBackground.cc

```diff
@@ -67,7 +67,7 @@ void LocalBackgroundAlgorithm::measure(afw::table::SourceRecord& measRecord,
     if (!psf) {
         throw LSST_EXCEPT(MeasurementError, NO_PSF.doc, NO_PSF.number);
     }
-    float const psfSigma = psf->computeShape().getDeterminantRadius();
+    float const psfSigma = psf->computeShape(psf->getAveragePosition()).getDeterminantRadius();
```

Comment:
This should use the position at the `center` of the footprint.  Though this will change the behavior, I believe the current behavior is incorrect.